src/interactions/panZoomInteraction.ts

```diff
@@ -93,12 +93,14 @@ export module Interactions {
       var newCornerDistance = this._cornerDistance();
 
       if (this._xScale != null && newCornerDistance !== 0 && oldCornerDistance !== 0) {
-        PanZoom._magnifyScale(this._xScale, oldCornerDistance / newCornerDistance, oldCenterPoint.x);
-        PanZoom._translateScale(this._xScale, oldCenterPoint.x - newCenterPoint.x);
+        var invertedPX = this._xScale.invert(oldCenterPoint.x);
+        this._xScale.zoomLevel(this._xScale.zoomLevel() * oldCornerDistance / newCornerDistance);
+        PanZoom._translateScale(this._xScale, this._xScale.scale(invertedPX) - newCenterPoint.x);
```

Comment:
`_magnifyScale(...)` used to take in the center of where the magnification should be.  With the center idea removed from the signature, the center is assumed to be the center of the scale range, meaning that a translation has to be done afterwards to get the scale position to be in the right spot.

Decently important note, based on the usage above + a bunch of searching online, the above is essentially the common solution for this type of problem